platform-includes/enriching-events/attach-screenshots/flutter.mdx

```diff
@@ -13,6 +13,32 @@ await SentryFlutter.init(
 );
 ```
 
+### Customize Screenshot Capturing
+
+<Note>
+
+Requires SDK version `8.11.0` or higher.
+
+</Note>
+
+Because capturing screenshots can be a resource-intensive operation on Flutter, it's limited to one screenshot every 2 seconds using a debouncing mechanism. This behavior can be overruled if you supply a `BeforeCaptureCallback` for screenshots in the `SentryFlutterOptions`.
+
+The `BeforeCaptureCallback` also allows you to customize the behavior based on event data, so you can decide when to capture a screenshot and when not to. For example, you can decide to only capture screenshots of fatal events:
+
+```java
+await SentryFlutter.init((options) {
+  options.attachScreenshot = true;
```

Comment:
```suggestion
```

I guess we can remove this as we can assume screenshots are already in place

Reply:
hmm there is a sample code above the beforeCapture though that shows how to enable screenshots

Reply:
Ah it's included, sry.